core-bundle/src/Controller/AbstractCustomBackendController.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Controller;
+
+use Contao\Ajax;
+use Contao\BackendMain;
+use Contao\BackendTemplate;
+use Contao\CoreBundle\Util\PackageUtil;
+use Contao\Environment;
+use Contao\Input;
+use Symfony\Component\HttpFoundation\Response;
+
+abstract class AbstractCustomBackendController extends AbstractController
+{
+    /**
+     * Render a Twig template with additional context for `@Contao/be_main`.
+     */
+    protected function render(string $view, array $parameters = [], Response $response = null): Response
```

Comment:
We should implement an `abstract function getResponse()` here as well.
This is why:
- You won't be able to render a Twig template decoupled from the `BackendMain` controller otherwise.
- Implementing `getResponse()` will be part of the contract, making it required to implement for developers. This is important because we must make sure the business logic in `BackendMain::__construct()` will be executed.

Reply:
Oh wait, this would disallow using multiple controllers per class, which is valid as well.